servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -2089,6 +2089,31 @@ public void testDiff() throws BaseNessieClientServerException {
             });
   }
 
+  @Test
+  public void commitLogExtendedForUnchangedOperation() throws Exception {
+    String branch = "commitLogExtended";
+    api.createReference()
+      .sourceRefName("main")
+      .reference(Branch.of(branch, null))
+      .create()
+      .getHash();
+    String head = api.getReference().refName(branch).get().getHash();
+    api.commitMultipleOperations()
+        .operation(Unchanged.of(ContentKey.of("key1")))
+        .commitMeta(CommitMeta.fromMessage("Commit #1"))
+        .branchName(branch)
+        .hash(head)
+        .commit();
+
+    List<LogEntry> logEntries =
+        api.getCommitLog().fetchAdditionalInfo(true).refName(branch).get().getLogEntries();
+
+    assertThat(
+            logEntries.size() == 1
+                && logEntries.get(0).getCommitMeta().getMessage().contains("Commit #1"))
+        .isTrue();
```

Comment:
Sorry about a late nitpick... I did not notice this before, but would it be clearer to use two `assertThat(...).isEqualTo(...)` statements instead of one `isTrue()` with an `AND` boolean construct? In the former case JUnit will produce more specific and easier to read failure messages, AFAIK.

Reply:
Could you also add asserts about `CommitLogEntry.getPuts()` and `getDeletes()`?

Reply:
It is ImmutableLogEntry  object, not CommitLogEntry. So, added a check that operations should be null.